notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        color = "background-color: lightgreen"
</pre><p>It is better to define this constant outside the function:</p><p><code>HIGHLIGHT_COLOR = "background-color: lightgreen"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
The constant was removed. It is obvious, that it is color by the value and looks like the constant doesn't make sense.